bonsai/ethereum-relay/src/uploader/completed_proofs/manager.rs

```diff
@@ -14,7 +14,10 @@
 
 use std::sync::Arc;
 
-use bonsai_proxy_contract::{Callback, ProxyContract};
+use bonsai_relay_contract::{
+    bonsai_relay::Callback as RelayCallback, bonsai_test_relay::Callback as TestRelayCallback,
```

Comment:
As types, these should be the same.

Reply:
Resolved in [4449c68](https://github.com/risc0/risc0/pull/738/commits/4449c68d50d47e296bfb42e62c650b97baf9593b)